client/packages/glsp-sprotty/src/utils/array-utils.ts

```diff
@@ -0,0 +1,34 @@
+
```

Comment:
I think we should add this class in lib/utils/ to be consistent with the structure in sprotty.